ktor-server/ktor-server-test-host/jvm/src/io/ktor/server/testing/Utils.kt

```diff
@@ -4,35 +4,48 @@
 
 package io.ktor.server.testing
 
-import io.ktor.application.*
 import io.ktor.http.*
+import org.junit.Assert.*
 
 /**
  * [on] function receiver object
  */
-public object On
+object On
 
 /**
  * [it] function receiver object
  */
-public object It
+object It
 
 /**
  * DSL for creating a test case
  */
 @Suppress("UNUSED_PARAMETER")
-public fun on(comment: String, body: On.() -> Unit) = On.body()
+fun on(comment: String, body: On.() -> Unit) = On.body()
 
 /**
  * DSL function for test test case assertions
  */
 @Suppress("UNUSED_PARAMETER")
-public inline fun On.it(description: String, body: It.() -> Unit) = It.body()
+inline fun On.it(description: String, body: It.() -> Unit) = It.body()
 
 /**
  * Return parsed content type from the test response
  */
-public fun TestApplicationResponse.contentType(): ContentType {
+fun TestApplicationResponse.contentType(): ContentType {
     val contentTypeHeader = requireNotNull(headers[HttpHeaders.ContentType])
     return ContentType.parse(contentTypeHeader)
 }
+
+
+inline fun <reified T : Throwable> assertFailsWith(block: () -> Unit) {
```

Comment:
Is it a public function? 
It could create confusion among the users of ktor-server-test-host who also use kotlin-test.

Reply:
sure, let me move it to the use places and make private